### PR TITLE
[fix](executor) Fe publish topic info tcp leak

### PR DESCRIPTION
**Pull request by whutpencil**
## Proposed changes

Issue Number: close #xxx

After publishing topic info to be, FE did not return the thrust client correctly, resulting in a TCP connection leak.

image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by wangbo — DISMISSED** [review]
LGTM

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/publish/TopicPublisherThread.java [hunk 1]
```diff
@@ -103,16 +103,25 @@ public TopicPublishWorker(TPublishTopicRequest request, Backend node, ResponseHa
...
                 client.publishTopicInfo(request);
+                ok = true;
                 LOG.info("publish topic info to be {} success, time cost={} ms",
                         be.getHost(), (System.currentTimeMillis() - beginTime));
             } catch (Exception e) {
                 LOG.warn("publish topic info to be {} error happens: , time cost={} ms",
                         be.getHost(), (System.currentTimeMillis() - beginTime), e);
             } finally {
+                if (ok) {
+                    ClientPool.backendPool.returnObject(address, client);
```
Comment (wangbo) [inline fe/fe-core/src/main/java/org/apache/doris/common/publish/TopicPublisherThread.java:121]: We'd better catch a throwable exception for returnObject/invalidateObject

**Review by wangbo — APPROVED** [review]
LGTM

**Comment by wangbo** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**